MSAL/src/util/ios/MSALRedirectUriVerifier.m

```diff
@@ -50,13 +51,13 @@ + (MSALRedirectUri *)msalRedirectUriWithCustomUri:(NSString *)customRedirectUri
     if (![NSString msidIsStringNilOrBlank:customRedirectUri])
     {
         NSURL *redirectURI = [NSURL URLWithString:customRedirectUri];
-
-        if (![self verifySchemeIsRegistered:redirectURI error:error])
+        
+        if (!bypassRedirectValidation && ![self verifySchemeIsRegistered:redirectURI error:error])
         {
             return nil;
         }
 
-        BOOL brokerCapable = [MSALRedirectUri redirectUriIsBrokerCapable:redirectURI];
+        BOOL brokerCapable = !bypassRedirectValidation && [MSALRedirectUri redirectUriIsBrokerCapable:redirectURI];
```

Comment:
bypassing of Redirect URI validation shouldn't impact whether URL is broker capable.

Reply:
I thought we designed to forward the request to broker only when the redirectURI is verified? I might misunderstand the purpose of verifying redirectURI in MSAL.

Reply:
My bad, I've misread the code :) So this is to ensure that we don't forward request to broker, unless redirect was validated.